content/about/mirrors.md

```diff
@@ -1,50 +1,50 @@
 ---
 title: "GRASS GIS Website mirrors"
-date: 2022-05-28T10:12:05+02:00
+date: 2025-05-11T10:12:05+02:00
 layout: "general"
 ---
 
-### Official main site via HTTP/FTP
+### Official main site
```

Comment:
In relation to adding page headers we may want to change the headings to h2 and subheadings to h3.